### PR TITLE
ammonite-repl 0.7

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/ammonite-repl.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  url "https://github.com/lihaoyi/Ammonite/releases/download/0.7.0/0.7.0", :using => :nounzip
+  sha256 "bcd86bee82b63f11bd4ae8b3c72b90401f3183b09f586326aeb6743f4c78f918"
 
   bottle :unneeded
 
   depends_on :java => "1.7+"
 
   def install
-    bin.install "ammonite-repl-0.6.2-2.11.8" => "amm"
+    bin.install Dir["*"].pop => "amm"
```
Comment (ilovezfs) [inline Formula/ammonite-repl.rb]: @DomT4 so just how much do we detest this?

Comment (DomT4) [inline Formula/ammonite-repl.rb]: It might do weird things if they ever add more files to the download, but that seems unlikely so eh.

Comment (ilovezfs) [inline Formula/ammonite-repl.rb]: It'll install the first non-dot file as "amm". So presumably the `test do` will die in a ball of :fire: if that's wrong for whatever reason.

Comment (DomT4) [inline Formula/ammonite-repl.rb]: I thought `.pop` returned the last item, so if they added a file called `bbb` that would be installed? Don't know. Maybe I'm misremembering Ruby, happens incredibly frequently.

Comment (ilovezfs) [inline Formula/ammonite-repl.rb]: You're right. `shift` does the first. We could do that instead. At this point there's only one file so they're the same.

Yep "bbb" would be installed as "amm" so `"#{bin}/amm -c 'print(\"hello world!\")'"` should not output "hello world!" if that's bogus.

Comment (DomT4) [inline Formula/ammonite-repl.rb]: `shift` probably makes sense then given the filename, but it seems likely either will keep working indefinitely unless upstream redesign the way they handle releases, and even then as you note the test will blow.

Comment (ilovezfs) [inline Formula/ammonite-repl.rb]: We could also be more careful and validate it's a jar file using `jar -tf` if you'd prefer.

Comment (DomT4) [inline Formula/ammonite-repl.rb]: I think it's alright. Let's not sex it up too much unless necessary.

Comment (ilovezfs) [inline Formula/ammonite-repl.rb]: Yeah I think `shift` should do the job fine for a while.

**Pull request closed without merging**